### PR TITLE
Revert "Create updates"

### PR DESCRIPTION
**Pull request by XDVelectical**
Reverts XDVelectical/PVP-kit#2

### CODE REVIEW

**Review by Velecticall — CHANGES_REQUESTED** [review]
Needs update

**Review by Velecticall — APPROVED** [review]

**Review by Velecticall — APPROVED** [review]

**Review by Velecticall — APPROVED** [review]
Done

**Review by Velecticall — APPROVED** [review]
Finished

File: updates [hunk 1]
```diff
@@ -1,2 +0,0 @@
-update here 
-new
```
Comment (XDVelectical) [inline updates:2]: better

**Review by Velecticall — APPROVED** [review]

**Review by Velecticall — APPROVED** [review]

**Review by Velecticall — COMMENTED** [review]
Done

**Pull request merged**